### DISCUSSION TITLE
Is there a component library for Remotion?

### DISCUSSION THREAD

**Discussion by BenjaminIsMyName** [remotion-dev/remotion#2947] [General]
React for the web has many component libraries, React Native has many component libraries... Is there ANY component library for Remotion?

I think something like ShadcnUI would be great. Some useful components could be:
- Code block
- Cursor (Click, double click)
- Toast 
- Animations (Fade in, fade out, slide up, slide down, and so on... Much like Reanimated)
- Typing effect (including typing sound?)
- Arrow 
- And so on...

I just saw this [(here)](https://twitter.com/delba_oliveira/status/1707439537054535867?t=nwjIEkf6xwBUMX-ZBHU4-g) video and I think it would take forever to recreate this without predefined components I can use to speed up the editing experience

**Comment by JonnyBurger** [comment]
Good idea!

Remotion components can always be published to GitHub/NPM, we collect all the l resources here: https://remotion.dev/resources

Of course we rely on growing a community to create a whole ecosystem of animated components.

Another step into this direction is that we are launching a marketplace for paid components next week. It will have little content at the beginning, but we are starting now.

**Comment by degueba** [comment]
A while back this thread asked for "a shadcn for Remotion." I wanted the same thing, so I built it.
Onda: 70 components plus 18 transitions, installed as source via npx ondajs add fade-in. The .tsx plus Zod schema land in your repo, yours to edit. There's an Interface set (terminals, code blocks, device frames) for product demos, plus title cards, charts, and <TransitionSeries> transitions. All from one motion language so it stays consistent.
Early and open source. Would love feedback from people who actually build with Remotion. 

Website: https://onda.video/
Github: https://github.com/degueba/onda
NPM: https://www.npmjs.com/package/ondajs

**Reply by JonnyBurger** [reply]
Added to `/docs/resources` for the next site update. Thanks for sharing Onda — the source-installed component library idea is excellent.
